local-development/docker-compose.yml

```diff
@@ -64,6 +64,7 @@ services:
     volumes:
       - fairspace-postgres-data:/var/lib/postgresql/data
     restart: unless-stopped
+    command: ["postgres", "-c", "checkpoint_timeout=43200"]
```

Comment:
this will work only for the local development, could you apply it also for Kubernetes deployment?